bangazonapi/views/customer.py

```diff
@@ -34,8 +34,8 @@ def update(self, request, pk=None):
             HTTP/1.1 204 No Content
         """
         customer = Customer.objects.get(user=request.auth.user)
-        customer.user.last_name = request.data["last_name"]
-        customer.user.email = request.data["email"]
+        customer.last_name = request.data["last_name"]
```

Comment:
This update method isn't working, but these lines of code actually shouldn't be changed. 

The last_name and email attributes are located on the user model, which can be accessed through the customer model via the 1:1 relationship. So if we want to set the value of the `last_name` and the `email` on the user, we've got to go _through_ the customer, like this: `customer.user.last_name` or  `customer.user.email`. 

To fix the update method, you'll need to examine your customer instance. Right now, every time a PUT request is sent to `customers/:id`, the authenticated user gets updated, regardless of the pk in the URL. How can you modify the query to `get` the customer from the db, where the pk matches the pk from the client?